pulsar-broker/src/test/java/org/apache/pulsar/broker/service/InactiveTopicDeleteTest.java

```diff
@@ -650,7 +650,6 @@ public void testBrokerDeleteInactivePartitionedTopicMetadataEnabled() throws Exc
         admin.brokers()
                 .updateDynamicConfiguration("brokerDeleteInactivePartitionedTopicMetadataEnabled",
                         "true");
-        Thread.sleep(2000);
         Awaitility.await().untilAsserted(()->{
             assertTrue(conf.isBrokerDeleteInactivePartitionedTopicMetadataEnabled());
         });
```

Comment:
I mean, use `atMost` method to control the timeout explicitly. See https://www.javadoc.io/doc/org.awaitility/awaitility/3.0.0/org/awaitility/Awaitility.html